ktor-jetty/src/org/jetbrains/ktor/jetty/CommandLine.kt

```diff
@@ -17,7 +15,9 @@ fun main(args: Array<String>) {
     }
 
     val jar = map["-jar"]?.let { File(it).toURI().toURL() }
-    val combinedConfig = ConfigFactory.load()
+    val argConfig = ConfigFactory.parseMap(map.filterKeys { it.startsWith("-P:") }.mapKeys { it.key.removePrefix("-P:") }, "Command-line options")
+    val combinedConfig = argConfig.withFallback(ConfigFactory.load())
+
```

Comment:
Because it should be way to override any config parameter via command-line. Personally I need it to override the http port as config file is under version control